tests/testSelectFluxMag0.py

```diff
@@ -35,18 +35,36 @@
 import lsst.afw.coord as afwCoord
 from lsst.obs.sdss.scaleSdssZeroPoint import ScaleSdssZeroPointTask
 from lsst.obs.sdss.selectFluxMag0 import SelectSdssFluxMag0Task
+import lsst.log
+
+logger = lsst.log.Log.getLogger("obs_sdss.testSelectFluxMag0")
+
+config = ScaleSdssZeroPointTask.ConfigClass()
+
+# Some of the tests require loading SDSS images from "lsst-db.ncsa.illinois.edu" and
+# require a login name and password. If the test is unable to connect to the external data,
+# some of the tests are skipped.
+noConnection = False
+try:
+    DbAuth.username(config.selectFluxMag0.host, str(config.selectFluxMag0.port))
+except Exception, e:
```

Comment:
The try...except block was originally in the `run` function before I updated the boilerplate. I modified it slightly but didn't pay attention to the except syntax.